### PR TITLE
docs: using `import` instead `require`

### PR DESCRIPTION
**Pull request by melonges**
in modern React tooling import statement is more preferable

### CODE REVIEW

File: APP.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ import {
 import { clusterApiUrl } from '@solana/web3.js';
 
 // Default styles that can be overridden by your app
-require('@solana/wallet-adapter-react-ui/styles.css');
+import('@solana/wallet-adapter-react-ui/styles.css');
```
Comment (glitch-txs) [inline APP.md:40]:
```suggestion
import '@solana/wallet-adapter-react-ui/styles.css';
```

**Comment by glitch-txs** [comment]
I don't have permissions to approve

**Comment by melonges** [comment]
@ryoqun @lorisleiva @yihau @2501babe

**Review by 2501babe — APPROVED** [review]

**Comment by 2501babe** [comment]
oh. i can approve in the code review sense but i also cant approve the ci run lol. probably this repo is controlled by @steveluscher ?

**Comment by steveluscher** [comment]
Thank you!

**Pull request merged**